### ISSUE TITLE
Suffix databinding

### ISSUE THREAD

**Issue by Liombe** [algolia/instantsearch-android#74]
Hello,

I'd like to report a minor error in BindingHelper.java

```/**
     * Gets the full version of an attribute, with its eventual prefix and suffix.
     *
     * @param view         the view mapped to this attribute.
     * @param rawAttribute the raw value of the attribute.
     * @return the attribute value, prefixed and suffixed if any was specified on the view.
     */
    public static String getFullAttribute(View view, String rawAttribute) {
        String attribute = "";
        final int id = view.getId();
        final String prefix = prefixes.get(id);
        final String suffix = suffixes.get(id);
        if (prefix != null) {
            attribute = prefix;
        }
        attribute += rawAttribute;
        if (suffix != null) {
            attribute = prefix;
        }
        return attribute;
    }
```

The suffix here is never added unfortunately nor the rawAttribute when the suffix is not null.
    
Here's an exemple

![image](https://user-images.githubusercontent.com/16174252/38241748-51c94064-3733-11e8-9883-38ee684e172c.png)

**Comment by PLNech** [comment]
Hi @Liombe, thanks for your report!
Wow, good catch. This is definitely a bug, I'll push a fix as soon as possible and will update you here!

**Comment by PLNech** [comment]
Hi @Liombe, this is just to notify you that the issue has been fixed and you can use versions [`v1.7.24`](https://github.com/algolia/instantsearch-android/releases/tag/1.7.24) and upwards which shipped the fix.
